src/main/java/at/hannibal2/skyhanni/data/BitsAPI.kt

```diff
@@ -0,0 +1,141 @@
+package at.hannibal2.skyhanni.data
+
+import at.hannibal2.skyhanni.events.InventoryFullyOpenedEvent
+import at.hannibal2.skyhanni.events.LorenzChatEvent
+import at.hannibal2.skyhanni.events.ScoreboardChangeEvent
+import at.hannibal2.skyhanni.utils.ItemUtils.getLore
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.NumberUtil.formatNumber
+import at.hannibal2.skyhanni.utils.StringUtils.matchMatcher
+import at.hannibal2.skyhanni.utils.StringUtils.matches
+import at.hannibal2.skyhanni.utils.StringUtils.trimWhiteSpaceAndResets
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+object BitsAPI {
+    var bits: Int
+        get() = ProfileStorageData.profileSpecific?.bits?.bits ?: 0
+        set(value) {
+            ProfileStorageData.profileSpecific?.bits?.bits = value
+        }
+    var currentFameRank: FameRank
+        get() = ProfileStorageData.profileSpecific?.bits?.currentFameRank ?: FameRank.NEW_PLAYER
+        set(value) {
+            ProfileStorageData.profileSpecific?.bits?.currentFameRank = value
+        }
+    var bitsToClaim: Int
+        get() = ProfileStorageData.profileSpecific?.bits?.bitsToClaim ?: 0
+        set(value) {
+            ProfileStorageData.profileSpecific?.bits?.bitsToClaim = value
+        }
```

Comment:
add a "private val dataStorageBits get() = ProfileStorageData.profileSpecific?.bits" makes it more readable (for the name of that Idk maybe find a better one then I gave)